Data-Structures/Graph/Graph2.js

```diff
@@ -21,42 +21,49 @@ class Graph {
     // initialize the adjacent list with a
     // null array
 
-    this.AdjList.set(v, [])
+    this.adjacencyList.set(v, [])
   }
 
   // add edge to the graph
-  addEdge(v, w) {
-    // get the list for vertex v and put the
-    // vertex w denoting edge between v and w
-    this.AdjList.get(v).push(w)
+  addEdge(vertex1, vertex2) {
```

Comment:
`v` and `w` are typical names for vertices. `vertex1` and `vertex2` is less readable.

Reply:
Would it be best to close this out and leave the file with naming as is? There could be better things to focus on with your time. I am game for that if you think all is good right now.

Reply:
It's not major, but it is a minor change for the worse, IMO.

Reply:
Ok. Thanks for the feedback @appgurueu